sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.dto.ts

```diff
@@ -120,3 +122,14 @@ export class CreateApplicationOfferingChangeRequestAPIInDTO {
   @MaxLength(REASON_MAX_LENGTH)
   reason: string;
 }
+
+/**
+ * Details to update the application offering change request by student.
+ */
+export class UpdateApplicationOfferingChangeRequestAPIInDTO {
```

Comment:
The name is too generic and may conflict with ministry on approving offering change.